README.md

```diff
@@ -35,6 +43,8 @@ import 'ember-tb-test-helpers/extend-qunit';
 * `findRole(role)` - Finds (with assert) an element with matching `[data-role]`
 * `fillInField(name, value)` - Fills in a field with a `[name]` with the given
   value
+* `within(scope, block)` - Scopes subsequent calls to
```

Comment:
I'm afraid that providing line numbers will quickly become out of sync.

What do you think about linking to the test file?